Apps/WebClient/src/ClientApp/src/components/modal/addQuickLink.vue

```diff
@@ -75,25 +85,28 @@ export default class AddQuickLinkComponent extends Vue {
 
     private get enabledQuickLinkFilter(): QuickLinkFilter[] {
         this.logger.debug(
-            `Enabled Quick Links - checkbox component key:  ${this.checkboxComponentKey}`
+            `Enabled Quick Link Filter - quick links:  ${JSON.stringify(
+                this.quickLinks
+            )}`
         );
-        if (this.checkboxComponentKey != 0) {
-            return this.enabledQuickLinks;
-        } else {
-            this.logger.debug(
-                `Enabled Quick Links - quick links:  ${this.quickLinks}`
-            );
-            if (this.quickLinks != undefined) {
-                this.populateEnabledQuickLinks();
-                const existingQuickLinks = this.quickLinks;
-                return this.getAvailabeQuickLinks(
-                    existingQuickLinks,
-                    this.enabledQuickLinks
-                );
-            }
-            this.populateEnabledQuickLinks();
-            return this.enabledQuickLinks;
+
+        // Only display quick links in filter that are enabled
+        this.enabledQuickLinks = this.quickLinkFilter.filter(
+            (filter: QuickLinkFilter) => filter.isEnabled
+        );
```

Comment:
Nit: the getter could be simplified further by moving the logic out of `this.getAvailabeQuickLinks()`:

```
        return this.quickLinkFilter.filter(
            (filter) =>
                filter.isEnabled &&
                this.quickLinks?.find(
                    (existingLink) =>
                        existingLink.filter.modules.length === 1 &&
                        existingLink.filter.modules[0] === filter.module
                ) === undefined
        );
```